standard/lexical-structure.md

```diff
@@ -787,14 +844,16 @@ The pre-processing directives provide the ability to skip conditionally sections
 
 > *Note*: The term "pre-processing directives" is used only for consistency with the C and C++ programming languages. In C#, there is no separate pre-processing step; pre-processing directives are processed as part of the lexical analysis phase. *end note*
 
+> *Note*: The pre-processor grammar, though defined using ANTLR lexical rules, does not define any lexical tokens in the language and is not “ANTLR ready”. Pre-processing involves processing the input and modifying the token stream produced by the lexer, this is not represented in this grammar and is part of compiler implementation and thus out of scope for this language specification. *end note*
```

Comment:
This is the big bit that hasn't been ANTLR-ised. I did look as to whether these rules could be treated as lexical ones which produce a single "PP Directive" token, but that is thwarted by the context-free specification...

Though I think I have a proposal as to how to handle this as any PR will build on this one and I've no idea how to do that in GitHub I've chosen to leave this until we've decided whether to accept this PR and its merged into the main draft-v6 branch. If you would like to know that proposal to help decide on *this* PR request it on the ECMA list and I'll explain once I've tested it.

Reply:
Suggest replacing the curly-double-quotes with straight ones. But other than that, I'm fine with the paragraph and your comment :)